index.html

```diff
@@ -63,14 +63,306 @@ <h1 class="hero-title">Want anything to be easy with LaslesVPN.</h1>
+            <h2 class="title-2 text-center">
+              Trusted by Thousands of Happy Customer
+            </h2>
+            <p class="text-center plain-text pb-60 m-0">
+              These are the stories of our customers who have joined us with
+              great pleasure when using this crazy feature.
+            </p>
+          </div>
+        </div>
+        <ul class="d-flex justify-content-between pb-50 m-0">
+          <li tabindex="0" class="testimonials-box">
+            <div class="d-flex justify-content-between align-items-center">
+              <img src="images/testimonials-img1.jpg" alt="" />
+              <p class="flex-direction-column d-flex">
+                <span class="title-3">Viezh Robert</span>
+                <span class="text-sm">Warsaw, Poland</span>
+              </p>
+              <p>
+                <span>4.5</span>
+                <span class="icon-star-filled testimonial-star"></span>
+              </p>
+            </div>
+            <p class="plain-text-black">
+              “Wow... I am very happy to use this VPN, it turned out to be more
+              than my expectations and so far there have been no problems.
+              LaslesVPN always the best”.
+            </p>
+          </li>
+          <li tabindex="0" class="testimonials-box">
+            <div class="d-flex justify-content-between align-items-center">
+              <img src="images/testimonials-img2.jpg" alt="" />
+              <p class="flex-direction-column d-flex">
+                <span class="title-3">Yessica Christy</span>
+                <span class="text-sm">Shanxi, China</span>
+              </p>
+              <p>
+                <span>4.5</span>
+                <span class="icon-star-filled testimonial-star"></span>
+              </p>
+            </div>
+            <p class="plain-text-black">
+              “I like it because I like to travel far and still can connect with
+              high speed.”.
+            </p>
+          </li>
+          <li tabindex="0" class="testimonials-box">
+            <div class="d-flex justify-content-between align-items-center">
+              <img src="images/testimonials-img3.jpg" alt="" />
+              <p class="flex-direction-column d-flex">
+                <span class="title-3">Kim Young Jou</span>
+                <span class="text-sm">Seoul, South Korea</span>
+              </p>
+              <p>
+                <span>4.5</span>
+                <span class="icon-star-filled testimonial-star"></span>
+              </p>
+            </div>
+            <p class="plain-text-black">
+              “This is very unusual for my business that currently requires a
+              virtual private network that has high security.”.
+            </p>
+          </li>
+        </ul>
+        <div class="d-flex justify-content-between align-items-center">
+          <div class="d-flex justify-content-between">
+            <div class="scroll-bullet"></div>
```

Comment:
I believe this should be a list of buttons. Use `ul` and `button`. Don't forget about `aria-label`